tests/requirements/dj40_cms311.txt

```diff
@@ -0,0 +1,4 @@
+-r base.txt
+
+Django>=4.0,<4.1
+django-cms>=3.11,<3.11
```

Comment:
```suggestion
django-cms>=3.11,<4.0
```

Reply:
Good catch ! @marksweb do you want I fixup the commit ?